samples/bot-all-cards/csharp/BotAllCards/Dialogs/AllCards.cs

```diff
@@ -0,0 +1,157 @@
+        {
+            var paths = new[] { ".", "Resources", "adaptiveCard.json" };
+            var adaptiveCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var adaptiveCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.adaptive",
+                Content = JsonConvert.DeserializeObject(adaptiveCardJson),
+            };
+
+            return adaptiveCardAttachment;
+        }
+
+        /// <summary>
+        /// Hero Card
+        /// </summary>
+        /// <returns></returns>
+        public static Attachment GetHeroCard()
+        {
+            var paths = new[] { ".", "Resources", "heroCard.json" };
+            var heroCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var heroCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.hero",
+                Content = JsonConvert.DeserializeObject(heroCardJson),
+            };
+
+            return heroCardAttachment;
+        }
+
+        /// <summary>
+        /// Thumbnail Card
+        /// </summary>
+        /// <returns></returns>
+        public static Attachment GetThumbnailCard()
+        {
+            var paths = new[] { ".", "Resources", "heroCard.json" };
+            var thumbnailCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var thumbnailCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.thumbnail",
+                Content = JsonConvert.DeserializeObject(thumbnailCardJson),
+            };
+
+            return thumbnailCardAttachment;
+        }
+
+        /// <summary>
+        /// Signin Card
+        /// </summary>
+        /// <returns></returns>
+        public static SigninCard GetSigninCard()
+        {
+            var signinCard = new SigninCard
+            {
+                Text = "BotFramework Sign-in Card",
+                Buttons = new List<CardAction>
+                {
+                    new CardAction(
+                        ActionTypes.OpenUrl, "Sign-in", value: "https://login.microsoftonline.com/") },
+            };
+
+            return signinCard;
+        }
+
+        /// <summary>
+        /// Card collections
+        /// </summary>
+        /// <returns></returns>
+        public static Attachment CollectionsCardAttachment()
+        {
+            var paths = new[] { ".", "Resources", "collectionsCard.json" };
+            var collectionsCardJson = File.ReadAllText(Path.Combine(paths));
+
+            var collectionsCardAttachment = new Attachment()
+            {
+                ContentType = "application/vnd.microsoft.card.adaptive",
+                Content = JsonConvert.DeserializeObject(collectionsCardJson),
+            };
+
+            return collectionsCardAttachment;
+        }
+
+        /// <summary>
+        /// Office 365 connector card
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Please add proper summary for this method.

`/// <returns></returns>`
Why this is empty whereas you are returning something?

Reply:
Fixed